lib/docker-sync/dependencies/docker.rb

```diff
@@ -0,0 +1,23 @@
+module DockerSync
+  module Dependencies
+    module Docker
+      DOCKER_NOT_AVAILABLE = 'Could not find docker binary in path. Please install it, e.g. using "brew install docker" or install docker-for-mac'.freeze
```

Comment:
this line is specific to osx.. should we refer user to `https://docs.docker.com/compose/install/` instead?